src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+/**
+ * Represents a Person's birthday in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBirthday(String)}
+ */
+public class Birthday {
+
+    public static final String MESSAGE_BIRTHDAY_CONSTRAINTS =
+            "Person names should only contain numbers and forward slashes,period or hash."
+                    + " The day month and year must be valid in form dd/mm/yyyy or dd/mm/yy";
+
+    public static final String BIRTHDAY_VALIDATION_REGEX = "^(?:(?:31(\\/|-|\\.)(?:0?[13578]|1[02]))\\1|(?:(?:29|30)"
```

Comment:
woah! :open_mouth: can this be simplified? 
(Actually no need to bother, in the interest of time!!)

Reply:
This is the only way to validate the dates, accounting for leap years/feb and all of that